File: cp_rust/src/bin/abc369-a-369.rs
// Created by Ayush Biswas at 2026/07/08 21:24
// https://atcoder.jp/contests/adt_easy_20250122_3/tasks/abc369_a
use cp_lib::*;

// @code begin
use cpio::*;

sol! {
    fn solution(
        [a, b]: [usize; 2]
    ) -> usize {
        if a == b {
            1
        } else if (a + b) % 2 == 0 {
            3
        } else {
            2
        }
    }
}
// @code end

File: cp_nim/src/bin/abc369-a-369.nim
# Created by Ayush Biswas at 2026/07/19 09:52
# https://atcoder.jp/contests/adt_easy_20250122_3/tasks/abc369_a
import ../lib/cpio

# @code begin

var r = newReader()

let (a, b) = r[int, int]

if a == b:
  echo 1
elif (a + b) mod 2 == 0:
  echo 3
else:
  echo 2

# @code end
